.github/workflows/pr-commands.yml

```diff
@@ -27,6 +27,17 @@ jobs:
           if [[ " ${allowlist[*]} " =~ " ${{github.event.sender.login}} " ]]; then
             echo "pass=1" >> "$GITHUB_OUTPUT"
           fi
+
+      - uses: actions/github-script@v6
+        with:
+          script: |
+            github.rest.reactions.createForIssueComment({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              comment_id: ${{ github.event.comment.id }},
+              content: 'eyes',
+            })
+
```

Comment:
add an improvement here. it looks like:
![Screenshot 2023-08-17 at 2 33 52 PM](https://github.com/solana-labs/move/assets/8209234/45cce53b-eec1-4249-b884-d804959dad17)

when the bot receives commands, it will send you an eyes emoji. 👀